docs/.vuepress/config.js

```diff
@@ -34,6 +34,7 @@ module.exports = {
           "/intro/sdk-design"
         ]
       },
+      // UNCOMMENT THIS IF YOU WANT TO BUILD DOCS : START
```

Comment:
we shouldn't comment anything to build the docs

Reply:
Right now docs are built by `cosmos/cosmos.network/Makefile` which downloads both Cosmos SDK Docs and SDK Application Tutorial. This is a **production scenario**.

I'm talking about **development scenario**, where you just want to run a simple command to view SDK Docs. And right now you do need to comment out these lines to run them locally.

I am advocating a move to a per-project build process for docs (see `updated-docs`), but we are not there yet. This is a workaround to adress the following PR:

https://github.com/cosmos/cosmos.network/pull/545

Reply:
should it say `COMMENT...` instead of `UNCOMMENT`then? Does the build process fail if you don't comment those lines? 

> I'm talking about development scenario, 

The comment should state that then: 

```
// comment the bracket below to only build the SDK docs locally
...
```
no `END` required